cli/src/main/java/com/devonfw/tools/ide/merge/xml/XmlMerger.java

```diff
@@ -119,6 +124,13 @@ public Document merge(XmlMergeDocument templateDocument, XmlMergeDocument worksp
       if (strategy == null) {
         strategy = XmlMergeStrategy.COMBINE; // default strategy used as fallback
       }
+      if (this.legacyXmlSupport && (templateRoot.lookupNamespaceURI("merge") == null)) {
```

Comment:
Is it enough to check for `null` or should we check if its not equal `MERGE_NS_URI`

Reply:
Actually in XML the namespace URL is the important thing and the `prefix` is just an alias to the URI.
So users could also name the prefix `merge2` or `ide`.
So Ideally you would do it the other way around:
```suggestion
      if (this.legacyXmlSupport && (templateRoot.lookupPrefix(MERGE_NS_URI) == null)) {
```